packages/server/graphql/mutations/helpers/updateReflectionLocation/removeReflectionFromGroup.ts

```diff
@@ -12,10 +12,30 @@ const removeReflectionFromGroup = async (reflectionId, {dataLoader}) => {
     .run()
   if (!reflection) throw new Error('Reflection not found')
   const {reflectionGroupId: oldReflectionGroupId, meetingId, promptId} = reflection
+  const oldReflectionGroup = await r
+    .table('RetroReflectionGroup')
+    .get(oldReflectionGroupId)
+    .run()
+  const {sortOrder: oldSortOrder} = oldReflectionGroup
+  const reflectionGroups = await r
+    .table('RetroReflectionGroup')
+    .getAll(meetingId, {index: 'meetingId'})
+    .filter({isActive: true})
+    .orderBy('sortOrder')
+    .run()
   const meeting = await dataLoader.get('newMeetings').load(meetingId)
 
   // RESOLUTION
-  const reflectionGroup = new ReflectionGroup({meetingId, promptId})
+  const nextReflectionGroupIndex =
```

Comment:
if the client lags, the oldReflectionGroup may already be inactive, which would result in `-1 + 1`, which would put this at the top (if the column has at least 1 card in it) instead of the bottom

Reply:
I don't quite follow on this one. Can you elaborate more? Appreciate it!

Reply:
`findIndex` returns -1 if no match is found.
that's possible because `oldReflectionGroupId` could refer to a reflection group that has already gone away